### DISCUSSION TITLE
Meeting Information

### DISCUSSION THREAD

**Discussion by JustFinchy** [o3de/sig-operations#15] [Q&A]
What times work best for everyone? 

Please leave your preferred days and times/time ranges that work for you. 

For example (Mine)
Wednesday - 12PM to 8PM EST
Thursday - 3PM to 8PM EST
Friday - 12PM to 4PM, 7PM to 8PM EST

Also Would like to see anyone who would be leading this comment.

**Comment by OBWANDO** [comment]
Wednesday 2PM to 6PM EST
Thursday 12PM to 6PM EST
Friday 12PM to 4PM EST

Keep in mind that the latter hours (past 1-2PM) can be more difficult on European members.

**Reply by JustFinchy** [reply]
yes, I can change and work around hours if needed. Just those are a good starting. 

Thursdays i usually do treatments on my muscle issues and Fridays I'm usually diving in the evening.
